### ISSUE TITLE
Scoring mechanism.

### ISSUE THREAD

**Issue by vaishnavm217** [louis-she/sfd.pytorch#13]
The score seems too strange to me.
some of them are 18-19 and others 8-6.
and the threshold has been kept to 6.
What is the score range?

**Comment by louis-she** [comment]
The score has been changed to softmax score, so the range is 0 to 1.

**Comment by louis-she** [comment]
This detector is in developing, the score threshold picking mechanism is not implemented yet. Now the score threshold is picked by hand. And we will add this soon and higher the mAP of this model.
